BW 1.5.2 release (#1034)
Concordium/concordium.github.io: source/mainnet/net/release-notes/release-notes-lp.rst

|bw|
-------------------------

>     June 20, 2024

Reviewer: Needs to be updated before we release.

Reviewer 2: ```suggestion
    June 24, 2024
```

>     Version 1.5.2 includes several changes and bug fixes.

Reviewer 2: @Ivan-Mahda was there a change regarding payload decoding for CIS3-standard?

Author: Added update to release notes

>         -   Added new option to edit account name. Name saved in local storage. Changed name displayed across all BrowserWallet.

Reviewer: I suggest not mentioning `local storage`. It seems an internal implementation detail.

```suggestion
        -   Added a new option to edit account names. The set names are used across the browser wallet for referring to the accounts.

```

>         -   Additional error message. Now instead of not showing invalid tokens, they displayed in token list with corresponding error. In order to show, that we found tokens in contract, but they have error.

Reviewer: I suggest starting with which part/flow of the browser wallet was improved so readers have an idea to which error messages you refer to.

E.g.
```
Improve error messages when searching for all tokenIDs in a given smart contract. Invalid tokenIDs in the contract are now displayed with their corresponding reason for being invalid. 
```

>         -   Display the optional name for ID providers when present, otherwise fallback to the chain name.

Reviewer: Sometimes the sentence starts with the past and sometimes with the present tense. Can we align it.

`Display`, `Remove`.

vs
`Added`, `Increased`.

>         -   Increased padding for QR code background. In dark mode, QR code not blending with background.

Reviewer: The second sentence could refer to the reason for the change (this seems to be the case here) or describe the state after the change. I suggest removing the second sentence since it might not be that important to know the exact reason (the reason for a small styling change is likely that it looked weird before).

```suggestion
        -   Increased padding of the QR code background element.
```
